synapse/res/templates/_base.html

```diff
@@ -13,13 +13,13 @@
 <body>
 <header class="mx_Header">
     {% if app_name == "Riot" %}
-        <img src="http://riot.im/img/external/riot-logo-email.png" width="83" height="83" alt="[Riot]"/>
+        <img src="https://riot.im/img/external/riot-logo-email.png" width="83" height="83" alt="[Riot]"/>
```

Comment:
Would it make sense to use `src="//riot.im/..."? If I'm not mistaken, this would make it work both for HTTP and HTTPS pages.

Reply:
Yeah, that would make the links protocol-relative and would work here too. But considering we don't have a reason to save https source loading, even on a http page (which most likely we don't have anymore) to save some cpu in tls handshake, I think its fine with https hardcoded too, now that its already done :)